pkgs/by-name/dy/dynamodb-local/package.nix

```diff
@@ -29,6 +36,29 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  passthru = {
+    tests.version = testers.testVersion {
+      package = dynamodb-local;
+    };
+    updateScript = writeShellScript "update-dynamodb-local" ''
+      set -o errexit
+      export PATH="${lib.makeBinPath [ curl jq yq common-updater-scripts ]}:$PATH"
+
+      NEW_VERSION=$(curl -s https://repo1.maven.org/maven2/com/amazonaws/DynamoDBLocal/maven-metadata.xml | xq -r '.metadata.versioning.latest')
```

Comment:
`xq` missing from dependencies?

Reply:
yq provides it
<img width="968" alt="image" src="https://github.com/NixOS/nixpkgs/assets/6224096/d868ec8b-4cb0-435e-a4c7-ae65be0c060d">

Reply:
TIL ! Thanks.